package.json

```diff
@@ -7,10 +7,10 @@
     "example": "examples"
   },
   "dependencies": {
-    "hls.js": "^0.5.44",
-    "video.js": "^5.13.0",
-    "videojs-contrib-hls": "^3.5.3",
-    "videojs-contrib-media-sources": "^3.1.5"
+    "hls.js": "^0.6.7",
+    "video.js": "^5.11.9",
```

Comment:
I was using the latest official release according to npm:
https://www.npmjs.com/package/video.js

And github:
https://github.com/videojs/video.js/releases